decidim/decidim
# Skip authenticity token in questionnaire templates

#### :tophat: What? Why?
Currently, is not possible as the preview form is being rendered inside the choose template form. This causes the authenticity_token to be rewriten, causing a 422 error. 

This PR allows you to create a new survey using templates. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?

#### Testing
1. Go  to Nightly, login as admin, visit any participatory space admin 
2. Create a new survey component 
3. Click on the newly created component, and choose a template 
4. Click Save, see there is a 422 error 
5. Apply patch 
6. See is working. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
